### PR TITLE
Update more packages for AT 12.0

### PR DESCRIPTION
**Pull request by tuliom**

### CODE REVIEW

**Comment by tuliom** [comment]
Build AT next SLES12 ppc64le 841 had a problem with the testsuite execution.  I requested a new build.

File: configs/12.0/packages/mpc/sources [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 ATSRC_PACKAGE_NAME="GNU MPC Library"
 ATSRC_PACKAGE_VER=1.1.0
-ATSRC_PACKAGE_REV=389c43dfbf14
+ATSRC_PACKAGE_REV=69ad1bcb8e9e
```
Comment (gftg85) [inline configs/12.0/packages/mpc/sources]: OK.

MPC doesn't have stable branches and this is the commit is the tag:

```
git describe --tags 69ad1bcb8e9e
1.1.0
```

File: configs/12.0/packages/mpfr/sources [hunk 2]
```diff
@@ -21,14 +21,14 @@
 
 ATSRC_PACKAGE_NAME="GNU MPFR Library"
 ATSRC_PACKAGE_VER=4.0.1
-ATSRC_PACKAGE_REV=12864
+ATSRC_PACKAGE_REV=12909
```
Comment (gftg85) [inline configs/12.0/packages/mpfr/sources]: OK
```
$ svn cat --revision 12909 https://scm.gforge.inria.fr/anonscm/svn/mpfr/branches/4.0/VERSION
4.0.2-dev
```

There are more updates to this branch, but they could be fetch in a new commit.

File: configs/12.0/packages/tcmalloc/sources [hunk 3]
```diff
@@ -21,7 +21,7 @@
 
 ATSRC_PACKAGE_NAME="TCMalloc"
 ATSRC_PACKAGE_VER=2.7
-ATSRC_PACKAGE_REV=f47a52ce85c3
+ATSRC_PACKAGE_REV=9608fa3bcf80
```
Comment (gftg85) [inline configs/12.0/packages/tcmalloc/sources]: OK.

tcmalloc doesn't provide stable branches and this commit is the same as the tag:

```
$ git describe --tags 9608fa3bcf80
gperftools-2.7
```

File: configs/12.0/packages/mpfr/sources (hunk 2 above)
Comment (tuliom) [reply configs/12.0/packages/mpfr/sources]: Indeed.  They're preparing to release version 4.0.2.
All the new updates appeared after I created this patch.

Comment (gftg85) [reply configs/12.0/packages/mpfr/sources]: Anyhow, I think this PR is good as is. (I hope I didn't give the impression that I don't).

**Comment by tuliom** [comment]
Build AT next SLES12 ppc64le 850 passed.  I'm merging this.

**Pull request merged**